src/shared/components/login.tsx

```diff
@@ -326,6 +326,7 @@ export class Login extends Component<any, State> {
               class="rounded-top img-fluid"
               src={this.captchaPngSrc()}
               style="border-bottom-right-radius: 0; border-bottom-left-radius: 0;"
+              alt={i18n.t('captcha')}
             />
             {this.state.captcha.ok.wav && (
               <button
```

Comment:
I don't know much about aria, but why doesn't this button need an aria-label? Is it because it has a title?

Reply:
Yeah exactly. Title isn't the absolute best way (that would be just having text inside the button) but it definitely works